### ISSUE TITLE
'something isnt playing nice' when opening settings

### ISSUE THREAD

**Issue by sgoranson** [joshhunt/GhostOverlay#29]
Hi, I was able to authorize, but when it loads it just says 'no tracked bounties' and a button to open settings. So I open settings, and just get 'something isnt playing nice'. Tried a few times and occasionally Game Bar crashes. 

I'm on Windows 10 insider edition, could that be an issue?

Are there logs anywhere that might be helpful? 

If you can point me at some basic compile/install instructions I can try to debug a little deeper. 

Great project, Thanks! -Steve

**Comment by Scottkwiggins** [comment]
@sgoranson 
Would you happen to have your Windows Display Scale set to anything other than "100% (Recommended)"? I have found that I have to switch back to 100% when I'm going to use the overlay to keep it working reliably.

**Comment by nelak2** [comment]
> @sgoranson
> Would you happen to have your Windows Display Scale set to anything other than "100% (Recommended)"? I have found that I have to switch back to 100% when I'm going to use the overlay to keep it working reliably.

Yes in fact I do. I have 4K monitors so I keep it around 125%. I'll keep your tip in mind.

Thanks

**Comment by bogus83** [comment]
Bump for this issue.  Please, PLEASE fix this so that all functions work correctly when Scale is something other than 100%.  It's a real pain to have to manually change the scale before and after every gaming session.

**Comment by ryanbowden** [comment]
@bogus83 Think this is fixed in the last release, working on my Surface go now set to 150%
